daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -7,15 +7,15 @@ description: "Release support and upgrade policies "
 ---
 
 ## Introduction
-This topic details the supported versions of Dapr releases, the upgrade policies and how deprecations and breaking changes are communicated.
+This topic details the supported versions of Dapr releases, the upgrade policies and how deprecations and breaking changes are communicated in all Dapr repositories (runtime, CLI, SDKs, etc) at versions 1.x and above.
 
 Dapr releases use `MAJOR.MINOR.PATCH` versioning. For example 1.0.0
 
-  * A `PATCH` version is incremented for bug and security hot fixes.
-  * A `MINOR` version is updated as part of the regular release cadence, including new features, bug and security fixes.
   * A `MAJOR` version is updated when there’s a non-backward compatible change to the runtime, such as an API change.  A `MAJOR` release can also occur then there is a considered a significant addition/change of functionality that needs to differentiate from the previous version.
+  * A `MINOR` version is updated as part of the regular release cadence, including new features, bug and security fixes.
+  * A `PATCH` version is incremented for bug and security hot fixes.
```

Comment:
Lets be clear on which types of bugs. Suggest to change to either P0 bugs or P0 issues.

Reply:
DOne.